examples/Exploring-different-models.ipynb

```diff
@@ -0,0 +1,802 @@
+{
```

Comment:
<p>Maybe this first para is just "First, import the libraries that we'll use in this notebook."</p><p> </p><p>(The para that introduces <code>import</code> statements is rarely a high information payload paragraph.)</p><p> </p><p>Or, if there's something to say about the use of TensorFlow, maybe add that.  I sincerely don't know, but I recall that models can work with Torch, so maybe you made a decision to use TensorFlow.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/256/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>